app/templates/email_validation.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:proactive:jobdescriptor:3.7"
+        xsi:schemaLocation="urn:proactive:jobdescriptor:3.7 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.7/schedulerjob.xsd"
+        name="Email Validation" projectName="Notifications"
+        priority="normal"
+        onTaskError="continueJobExecution"
+        maxNumberOfExecution="2"
+>
+  <description>
+    <![CDATA[ Workflow to pause the job and send a validation email ]]>
+  </description>
+  <taskFlow>
+    <task name="EmailValidation"
+    
+    
+    onTaskError="pauseJob" >
+      <description>
+        <![CDATA[ Task to pause the job and send a validation email ]]>
+      </description>
+      <scriptExecutable>
+        <script>
+          <code language="python">
+            <![CDATA[
+# Please fill variables
+
+notification_message = 'Put your email validation message here'
+email_to = 'user@domain.org'
+user_to_name = 'user'
```

Comment:
Maybe these variables can be set up as task variables, as it can be more user-friendly